tests/python/TestVideos.py

```diff
@@ -96,6 +96,29 @@ def test_addVideo(self):
         for i in range(0, number_of_inserts):
             self.assertEqual(response[i]["AddVideo"]["status"], 0)
 
+    def test_extractKeyFrames(self):
+
+        db = self.create_connection()
+
+        fd = open("../../tests/videos/Megamind.mp4", 'rb')
+        video_blob = fd.read()
+        fd.close()
+
+        props = {}
+        props["name"] = "video_test_kf"
+
+        video_params = {}
+        video_params["extract_kf"] = True
+        video_params["properties"] = props
+        video_params["codec"]      = "h264"
+
+        query = {}
+        query["AddVideo"] = video_params
+
+        response, obj_array = db.query([query], [[video_blob]])
+
+        self.assertEqual(response[0]["AddVideo"]["status"], 0)
```

Comment:
it would be nice to query for VD:KF and check we are getting the right number of i frames, as part of the same test.